src/components/MessageList/__tests__/MessageList.test.tsx

```
@@ -0,0 +1,636 @@
+const TestWrapper = ({ children }: { children: ReactNode }) => (
+  <BrowserRouter>
+    {children}
+  </BrowserRouter>
+);
+
+// Helper function to create mock messages
+const createMockMessage = (
+  id: string,
+  text: string,
+  type: MessageType = 'chat',
+  uid: string = 'user1',
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)